rollup.config.js

```diff
@@ -0,0 +1,19 @@
+import nodeResolve from '@rollup/plugin-node-resolve';
+
+export default {
+	input: {
+		'd2lfetch-auth': './src/unframed/index.js',
+		'd2lfetch-auth-framed': './src/framed/index.js'
+	},
+	output: {
+		dir: 'es6',
+		entryFileNames: '[name].js',
+		format: 'esm',
+		sourcemap: true
+	},
+	plugins: [
+		nodeResolve({
+			browser: true
+		})
+	]
+};
```

Comment:
This is dramatically simplified. Technically I don't think we actually need a Rollup step anymore since everything in this project is now ESM. Unfortunately, some folks are importing from `d2l-fetch-auth/es6/*` (and from the `src` directory) so we can't quite get rid of it just yet. If those could be cleaned up, then Rollup could go away.